cron/poll_read_issue_comments.py

```diff
@@ -35,13 +36,31 @@ def get_seconds_remaining(api, comment_id):
     return seconds_remaining
 
 
+def insert_or_update_issue(api, issue_id, number):
+    # get more info on the issue
+    gh_issue = gh.issue.open_issue(api, settings.URN, number)
+
+    # get user from db
+    user, _ = User.get_or_create(user_id=gh_issue["user"]["id"],
+                                 defaults={"login": gh_issue["user"]["login"]})
+
+    # db insert
+    issue, _ = Issue.get_or_create(issue_id=issue_id,
+                                   number=number,
+                                   user=user,
+                                   created_at=gh_issue["created_at"],
+                                   expedited=False,
+                                   is_pr="pull_request" in gh_issue)
+
+    return issue
+
+
 def insert_or_update(api, cmd_obj):
     # Find the comment, or create it if it doesn't exit
     comment_id = cmd_obj["global_comment_id"]
-    issue, _ = Issue.get_or_create(issue_id=cmd_obj["issue_id"])
     user, _ = User.get_or_create(user_id=cmd_obj["user"]["id"],
                                  defaults={"login": cmd_obj["user"]["login"]})
-
+    issue = insert_or_update_issue(api, cmd_obj["issue_id"], cmd_obj["number"])
```

Comment:
Yep. Read comments.py / get_all_issue_comments. It doesn't return the entire structure. You might need to change that